token.go

```diff
@@ -733,15 +750,19 @@ func processSingleResponse(ctx context.Context, sess *tdsSession, ch chan tokenS
 				}
 				return
 			}
-			if sess.logFlags&logRows != 0 && done.Status&doneCount != 0 {
-				sess.logger.Log(ctx, msdsn.LogRows, fmt.Sprintf("(%d row(s) affected)", done.RowCount))
-			}
 			ch <- done
 			if done.Status&doneCount != 0 {
-				if outs.msgq != nil {
-					_ = sqlexp.ReturnMessageEnqueue(ctx, outs.msgq, sqlexp.MsgRowsAffected{Count: int64(done.RowCount)})
+				if sess.logFlags&logRows != 0 && done.Status&doneCount != 0 {
```

Comment:
Is this condition still needed "done.Status&doneCount != 0", since it is checked above?